server/server/data_loader/textfunctions.py

```diff
@@ -0,0 +1,38 @@
+import regex
+import unicodedata
+
+def pali_sort_key(input, _charvalue = {}):
```

Comment:
don't initialize mutable objects in a function definition. It can easily lead to very strange and hard to debug bugs